res/houdini/MainMenuCommon.XML

```diff
@@ -3,6 +3,14 @@
     <menuBar>
         <subMenu id="avalon_menu">
             <label>Avalon</label>
+            <scriptItem id="avalon_context">
+                <labelExpression><![CDATA[
+from avalon import api
+return "%s - %s" % (api.Session["AVALON_ASSET"], api.Session["AVALON_TASK"])
+]]></labelExpression>
+                <scriptCode><![CDATA[return None]]></scriptCode>
```

Comment:
This line `<scriptCode><![CDATA[return None]]></scriptCode>` will raise Error, you could just remove it since `<scriptCode>` is not a must for `<scriptItem>`.

Reply:
fixed